src/components/modal.js

```diff
@@ -0,0 +1,166 @@
+(() => ({
+  name: 'Modal',
+  type: 'CONTAINER_COMPONENT',
+  allowedTypes: ['BODY_COMPONENT', 'CONTENT_COMPONENT'],
+  orientation: 'HORIZONTAL',
+  jsx: (
+    <div className={B.env === 'dev' ? classes.panel : ''}>
```

Comment:
Why do we always have a wrapper DIV? And why is the className `panel` only used in the page builder, not in runtime? Because there is now a lot of styling attached that has no purpose.

Reply:
Remember we only need this extra DIV in the pagebuilder for having the drag and drop events attached.

Reply:
No specific reason. I'm copy/pasting and learning along the way, so thanks for the tips 👍